src/clients/rpc_tls_client.h

```diff
@@ -175,15 +183,16 @@ class HttpRpcTlsClient : public TlsClient, public http::ResponseProcessor
       method,
       {body.data(), body.size()},
       http::headervalues::contenttype::MSGPACK,
-      verb);
+      verb,
+      jwt);
```

Comment:
Seems like the id_token (jwt) is not being set with Bearer scheme? Also, where do you specify that this belongs to Authorization header?